generate.xml

```diff
@@ -1875,6 +1883,9 @@
         <library name="bitcoin-protocol" />
         <library name="bitcoin-node" />
         <headers path="include" />
+        <!-- for mbedtls and mongoose C project support -->
+        <headers path="src/external/mbedtls/include" />
+        <option value="CFLAGS='-std=c99 -Wno-pedantic -Wno-format -Wno-comment -Wno-long-long -Wno-overlength-strings'" />
```

Comment:
This seems pretty onerous to apply to the full server build. Is it the case that these *only* affect the build of .c files?

Reply:
Yes, I tested this many times in order to get right.  It only affects CFLAGS and not the CPPFLAGS.

Reply:
Cool